apps/api/v2/src/ee/event-types/event-types_2024_04_15/inputs/update-event-type.input.ts

```diff
@@ -177,6 +177,14 @@ export class BookingField_2024_04_15 {
   @ValidateNested({ each: true })
   @Type(() => Source)
   sources?: Source[];
+
+  @IsOptional()
+  @IsNumber()
+  maxLength?: number;
+
+  @IsOptional()
+  @IsNumber()
+  minLength?: number;
```

Comment:
we probably also need this in packages/platform/types/event-types/event-types_2024_06_14/inputs/booking-fields.input.ts

Reply:
@supalarry